app/controllers/v0/profile/personal_informations_controller.rb

```diff
@@ -7,22 +7,13 @@ class PersonalInformationsController < ApplicationController
 
       # Fetches the personal information for the current user.
       # Namely their gender and birth date.
-      #
-      # @return [Response] Sample response.body:
-      #   {
-      #     "data" => {
-      #       "id"         => "",
-      #       "type"       => "mvi_models_mvi_profiles",
-      #       "attributes" => {
-      #         "gender"     => "M",
-      #         "birth_date" => "1949-03-04"
-      #       }
-      #     }
-      #   }
-      #
       def show
-        response = @current_user.mpi.profile
-
+        response = OpenStruct.new({
+                                    'id': @current_user.account_uuid,
+                                    'type': 'mvi_models_mvi_profiles',
```

Comment:
Setting `type` here doesn't do anything, it gets overridden when serialized (it'll be `open_structs` to match the type of the class that we serialize). There are ways we can fix it, but I don't think the frontend cares what the `type` is here, so I think we should just remove this line and let the type be `open_structs`)